packages/core/src/classes/StorageManager.ts

```diff
@@ -29,45 +32,52 @@ export class StorageManager<D extends IStorable = IStorable> {
 
   addItem(item: D): string | undefined {
     const id = item.getId()
-    if (!this.hasItem(id)) {
+    if (this.hasItem(id)) {
```

Comment:
if hasItem we return undefined
are you sure?

Reply:
Yep
If we have item by his id, we don't add it, we shoul update it